Examples/Image/GettingStarted/07_Deconvolution_Visualizer.py

```diff
@@ -11,7 +11,7 @@
 from cntk.ops import combine
 from cntk.io import MinibatchSource, CTFDeserializer, StreamDef, StreamDefs
 from PIL import Image
-from cntk import graph
+from cntk.logging import graph
```

Comment:
Rebased. The only remaining change to this file deals with the indexing change discussed above.